cmd/kubeadm/app/cmd/init.go

```diff
@@ -140,6 +140,9 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 				kubeadmutil.CheckErr(err)
 			}
 
+			// Downcase SANs. Some domain names (like ELBs) have capitals in them.
+			configutil.LowercaseSANs(externalcfg.APIServerCertSANs)
```

Comment:
This must be called by `configutil.SetInitDynamicDefaults` instead of here to be run every time we load a config instead of just for `kubeadm init`